jquery.hotspot.js

```diff
@@ -310,6 +310,10 @@
 
 		if (this.config.interactivity === 'click') {
 			spot.on('click', function(event) {
+				// Handle User-Defined onclick event, if defined
+				if(widget.config.hasOwnProperty('onclick')) {
+                                    widget.config.onclick(hotspot)
+				}
```

Comment:
@RogerWebb It's a nice feature to have. But what's the plan for `spot_html` element when `onclick` is defined in the widget config. I think it should also get handled.